app/src/main/java/com/antosik/benchproject/app/popular/movies/view/PopularMoviesRecyclerAdapter.kt

```diff
@@ -19,7 +19,10 @@ class PopularMoviesRecyclerAdapter(private val moviesViewModel: PopularMoviesVie
         }
     }
 
-    override fun setViewHolder(parent: ViewGroup, viewType: Int): BaseViewHolder<Movie> = PopularMoviesHolder(
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BaseViewHolder<Movie> =
+        setViewHolder(parent)
+
+    private fun setViewHolder(parent: ViewGroup): BaseViewHolder<Movie> = PopularMoviesHolder(
         PopularMovieRecyclerItemLayoutBinding.inflate(LayoutInflater.from(parent.context), parent, false)
```

Comment:
Could be inlined to not create private function with just one function invocation

Reply:
Done 😄